build.gradle

```diff
@@ -17,6 +19,9 @@ buildscript {
 allprojects {
     repositories {
         maven { url 'https://nexus.k40s.net/repository/maven-public/' }
+        maven { url 'https://jitpack.io' }
```

Comment:
Apparently you noticed that my Nexus is down. Sorry for that. 

I am currently rebuilding my infrastructure.